doc/minetestserver.6

```diff
@@ -1,4 +1,4 @@
-.\" Minetestserver man page
+.\" Minetest-server man page
```

Comment:
No, it's about the Minetest server-only binary, although its identifier is currently `minetestserver`.